README.md

```diff
@@ -27,52 +27,53 @@ delivered in each release and our future plans.
 
 ## Egeria governance
 
-This project aims to operate in a transparent, accessible way for the benefit
-of the Egeria community.
-All participation in this project is open and not
-bound to any corporate affiliation.
+This project aims to operate in a transparent, accessible way for the benefit of the Egeria community.
+All participation in this project is open and not bound to any corporate affiliation.
 
-To understand how to join and contribute see the 
+To understand how to join and contribute, see the 
 [Community Guide](./Community-Guide.md).
 
-All participants are bound the ODPi's [Code of Conduct](https://github.com/odpi/specs/wiki/ODPi-Code-of-Conduct).
+All participants are bound to the ODPi's [Code of Conduct](https://github.com/odpi/specs/wiki/ODPi-Code-of-Conduct).
 The governance of the project is described in more detail in the
 [Egeria Operations Guide](./Egeria-Operations.md).
 
 ## Call schedule
 
-The Egeria team run three types of calls and all are welcome
+The Egeria team runs three types of calls and all are welcome.
 
-* **Developers call** - one hour - 
-  Discussion on code development - not minuted -
-  
-  10 am (UK); 11 am (Amsterdam/CET); 12pm (Bucharest)
-  
-  Zoom Conference [https://zoom.us/j/523629111](https://zoom.us/j/523629111)
-  
-* **Community call** - one hour - 
-  Demos, meet-ups and other activities going on in the Egeria community
-  
-  8am (EST); 1pm (UK/GMT); 2pm (Amsterdam/CET); 3pm (Bucharest)
-  
-  Zoom Conference [https://zoom.us/j/523629111](https://zoom.us/j/523629111)
- 
-  Meeting Agenda and Minutes are here [https://wiki.lfaidata.foundation/display/EG/Community+Meetings](https://wiki.lfaidata.foundation/display/EG/Community+Meetings)
-  
-* **Technical Steering Committee (TSC)** - one hour -
-  Discussion on the management of the community
-  
-  Activities and call schedule
-  are here [https://wiki.lfaidata.foundation/display/EG/Egeria+Technical+Steering+Committee+%28TSC%29+Home](https://wiki.lfaidata.foundation/display/EG/Egeria+Technical+Steering+Committee+%28TSC%29+Home)
+Calls are scheduled for one hour on Zoom Conference [https://zoom.us/j/523629111](https://zoom.us/j/523629111).
 
+[Egeria Calendar](https://lists.lfaidata.foundation/g/egeria-technical-discuss/calendar) holds the latest timings and information for these calls.
+
+* **Community call** 
+   * Every Wednesday.
+   * Demos, meet-ups and other activities going on in the Egeria community.
+   * *Egeria Community Call (every Wednesday)* in the
+   [Egeria Calendar](https://lists.lfaidata.foundation/g/egeria-technical-discuss/calendar).    
+   * Zoom Conference [https://zoom.us/j/523629111](https://zoom.us/j/523629111).
```

Comment:
Do we also want to have the zoom link per event? If so lest add it also for the other two calls (or just remove it here and have it on top).

Reply:
I think this is done with the latest update